release-notes/release-notes521.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: default
+title: TinyMCE 5.2.1
+title_nav: TinyMCE 5.2.1
+description: Release notes for TinyMCE 5.2.1
+keywords: releasenotes bugfixes
+---
+
+## Overview
+
+These release notes provide an overview of the changes for {{site.productname}} 5.2.1, including:
+
+- [General bug fixes](#generalbugfixes)
+- [Accompanying Premium Plugin changes](#accompanyingpremiumpluginchanges)
+- [Upgrading to the latest version of TinyMCE 5](#upgradingtothelatestversionoftinymce5)
+
+> This is the {{site.cloudname}} and {{site.enterpriseversion}} release notes. For information on the latest community version of {{site.productname}}, see: [{{site.productname}} Changelog]({{site.baseurl}}/changelog/).
+
+## General bug fixes
+
+{{site.productname}} 5.2.1 provides fixes for the following bugs:
+
+* Fixed the "is decorative" checkbox in the image dialog clearing after certain editor events.
+* Fixed possible uncaught exception when a `style` attribute is removed using a content filter on `setContent`.
+* Fixed the table selection not functioning correctly in Microsoft Edge 44 or higher.
```

Comment:
"Fixed the table selection" -> "Fixed table selection"?

Reply:
fixed by https://github.com/tinymce/tinymce-docs/pull/1442/commits/5238ea966695fe907032e126e5e5cb28ce94b6e8